library/src/test/java/org/apache/apex/malhar/lib/state/managed/DefaultBucketTest.java

```diff
@@ -227,4 +233,33 @@ public void testFreeMemory() throws IOException
 
     testMeta.defaultBucket.teardown();
   }
+
+  @Test
+  public void testBloomFilter() throws IOException
```

Comment:
This test would succeed whether or not bloom filter is enabled. Can we do some sort of validation whether the bloom filter in the bucket is consistent with the results?